releases/UWP/README.md

```diff
@@ -60,11 +60,12 @@ When using Visual Studio this package requires Visual Studio 2015 Update 3 or la
     - System.Threading.Overlapped
     - System.Xml.XmlDocument
 
-### 5.2.2 (July 14, 2016)
-When using Visual Studio this package requires Visual Studio 2015 Update 3 or later.
+### 5.2.2 (June 27, 2016)
+When using Visual Studio this package requires Visual Studio 2015 Update 3 or later. You can read more about these changes in the Visual Studio 2015 Update 3 blog [post](https://www.visualstudio.com/en-us/news/releasenotes/vs2015-update3-vs).
 
-- .NET Native 1.4.
-- Universal Windows Platform 5.2.2 is the default in-box reference in Visual Studio 2017.
+- Fixed several customer reported bugs.
+- Improves release build compilation times of large applications.
```

Comment:
could you make these sentences past tense to match the rest of the doc?  otherwise LGTM.